components/TimeStackedBarChart.vue

```diff
@@ -81,13 +111,17 @@ export default {
       if (this.dataKind === 'transition') {
         return {
           lText: this.sum(this.pickLastNumber(this.chartData)).toLocaleString(),
-          sText: `${this.labels[this.labels.length - 1]} の合計`,
+          sText: `${this.$t(this.$t("{MM/DD}の合計"), {
```

Comment:
`this.$t`がひとつ多いです
また、`<script>`内ではstringにはシングルクォーテーションを使ってください
```suggestion
          sText: `${this.$t('{MM/DD}の合計', {
```